README.md

```diff
@@ -1,2 +1,91 @@
-# neo3-boa
-write smart contracts for Neo3 in python
+<h1 align="center">neo3-boa</h1>
+<p align="center">
+  Write smart contracts for Neo3 in Python by COZ
+</p>
+
+
+
+- [Overview](#overview)
+
+## Overview
+
+The `neo3-boa` compiler is a tool for compiling Python files to the `.nef` and `.manisfest.json` formats for usage in the [Neo Virtual Machine](https://github.com/neo-project/neo-vm/) which is used to execute contracts on the [Neo Blockchain](https://github.com/neo-project/neo/).
+
+
+#### What does it currently do
+
+- Compiles a subset of the Python language to the `.nef` and `.manisfest.json` format for use in the [Neo Virtual Machine](https://github.com/neo-project/neo-vm)
+
+- Works for Python 3.6+
+
+- Convert Functions
+
+- Convert Local Variable Declarations and Assignments 
+    
+    ```
+    foo: int = 42
+    bar = foo
+    ```
+- Convert Number Arithmetic Operations (`+`, `-`, `*`, `/`, `//`, `%`, `**`)
```

Comment:
Neo3-boa doesn't support `/` and `**` yet. Could you change those two operations to 'What will it do'?

Reply:
Ok! Sorry, I saw all of those in the examples and thought it was already implemented.
I'll fix it.